invenio_app_ils/search/api.py

```diff
@@ -32,32 +32,6 @@ def search_by_pid(self, *pids):
         """Retrieve documents with the given pid(s)."""
         return self.filter("terms", pid=pids)
 
-    def search_by_series_pid(self, series_pid=None):
```

Comment:
we will not search by series pid?

Reply:
It is not needed any more normally. This was a leftover from a resolver, that was resolving series from docs. But now they are handled with relations, so dead code